tests/blackbox/stratisd_cert.py

```diff
@@ -20,12 +20,41 @@
 import time
 import unittest
 
-from testlib.utils import exec_command, exec_test_command, process_exists
 from testlib.dbus import StratisDbus, clean_up
+from testlib.stratis import STRATIS_CLI
+from testlib.utils import exec_command, exec_test_command, process_exists, p_n, fs_n
 
 DISKS = []
 
 
+def make_test_pool():
+    """
+    Create a test pool that will later get destroyed
+    :return: Name of the created pool
+    """
+    pool_name = p_n()
+    assert (exec_test_command([STRATIS_CLI, "pool", "create", pool_name, DISKS[0]]))[
```

Comment:
You want to avoid using the CLI anywhere in these stratisd tests. So you'll want to find a different way to define ```make_test_pool``` and ```make_test_filesystem``` which uses ```StratisDbus``` methods.